components/Comments/Comment.vue

```diff
@@ -160,6 +187,37 @@
       }
     },
     mounted () {
+      const el = this.$el.getElementsByClassName('comment-main')[0]
+
+      if (this.depth > 0) {
+        el.classList.remove('depth-zero')
```

Comment:
This code is really hard to maintain, isn't there a VueJS way to do it? Feel free to add more components.

Reply:
It's replaced with a ref